bin/test-lxd-devlxd-vm

```diff
@@ -141,10 +141,25 @@ lxc stop -f v1
 sleep 5
 [ "$(lxc config get v1 volatile.last_state.ready)" = "false" ]
 
+# Test nested VM functionality.
+lxc start v1
+waitVMAgent v1
+
+lxc exec v1 -- apt-get install snapd --yes
+lxc exec v1 -- snap install lxd --channel=latest/edge
+lxc exec v1 -- /snap/bin/lxd init --auto
+lxc exec v1 -- /snap/bin/lxc launch images:ubuntu/22.04/cloud v1v1 --vm
+sleep 30
+lxc exec v1 -- /snap/bin/lxc info v1v1 | grep -F RUNNING
```

Comment:
Instead of calling `lxc exec` multiple times, why not do something like this?

```
cat <<- EOF | lxc exec v1 -- sh
apt-get install snapd --yes
snap install lxd --channel=latest/edge
/snap/bin/lxd init --auto
/snap/bin/lxc launch images:ubuntu/22.04/cloud v1v1 --vm
sleep 30
/snap/bin/lxc info v1v1 | grep -F RUNNING
EOF
```